.github/workflows/publish-docs.yml

```diff
@@ -16,15 +16,27 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
```

Comment:
I've moved the `uses: actions/checkout@...` and `name: checkout submodules` steps to the top of the steps list.